src/components/styles/Navigation.css

```diff
@@ -1,21 +1,64 @@
 .nav {
   display: flex;
-  align-items: center;
-  justify-content: center;
-  gap: 7%;
-  padding: 4%;
-  background-color: rgb(204, 204, 204);
+  border-bottom: 1px solid #b7b8bd;
+  background-color: white;
 }
 
 .nav ul {
   display: flex;
-  gap: 12%;
+  gap: 40%;
+  padding-top: 2%;
+  padding-left: 0;
 }
 
-.nav ul li {
+li,
+a {
   list-style-type: none;
+  text-decoration: none;
+  color: #121212;
+  font-size: 20px;
 }
```

Comment:
- [ ] Please 🙏🏽 correct the styles according to the font family specifications to match the Zeplin design.

![image](https://github.com/Atifzada/BookStore_App/assets/58642949/a5b37992-32b1-4e18-bafd-b8ef041added)